src/main/java/link/infra/borderlessmining/config/ConfigScreen.java

```diff
@@ -37,7 +37,7 @@ protected final void init() {
 		entries = new ConfigListWidget(client, width, height, 32, height - 32, 25);
 		addElements();
 		addDrawableChild(entries);
-		addDrawableChild(new ButtonWidget(width / 2 - 100, height - 27, 200, 20, ScreenTexts.DONE, (button) -> {
+		addDrawableChild(new UnprotectedButtonWidget(width / 2 - 100, height - 27, 200, 20, ScreenTexts.DONE, (button) -> {
```

Comment:
Use `ButtonWidget.builder` instead of creating a new class

Reply:
Done.